strings/src/main/res/values/strings.xml

```diff
@@ -1159,4 +1159,23 @@
     -->
     <string name="open_settings">Open Settings</string>
 
+    <!--
+    ##############################################
+    # Crash handling
+    ##############################################
+    -->
+
+    <string name="crash_last_run">There was a crash last time the app was running!</string>
```

Comment:
I'm not particularly happy with the copywriting here. @lognaturel can you suggest some alternatives?